ztp/source-crs/MachineConfigAcceleratedStartup.yaml

```diff
@@ -37,6 +59,22 @@ spec:
           Type=simple
           ExecStart=/usr/local/bin/accelerated-container-startup.sh
 
+          # Maximum wait time is 600s = 10m:
+          Environment=MAXIMUM_WAIT_TIME=600
+
+          # Steady-state threshold
+          # Allowed values:
+          #  4  - absolute pod count (+/-)
+          #  4% - percent change (+/-)
+          #  -1 - disable the steady-state check
+          Environment=STEADY_STATE_THRESHOLD=-1
+
+          # Steady-state window = 60s
+          # If the running pod count stays within the given threshold for this time
+          # period, return CPU utilization to normal before the maximum wait time has
+          # expires
+          Environment=STEADY_STATE_WINDOW=60
```

Comment:
What does the steady state window do if the threshold is set to -1?

Reply:
It's in the comment right above [Line 69](https://github.com/openshift-kni/cnf-features-deploy/pull/699/files/ce1a4ae9d1a944de88dfe39d0a476a0cbe619033#diff-1abecdd36031dcbb754c51b436a04ea54096047e5992dfc007a903ec74f70b43R69)
```
          #  -1 - disable the steady-state check
```

Reply:
So the whole check is disabled, and this is ignorend.  It's just in the file here for boiler-plate if you ever set the threshold to >=0

Reply:
Ah, so just there for reference.